src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_RER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCONT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCONT = 20;
+
+  if (days >= LONG_TERM) {
+    return days * PRICE_RER_DAY - LONG_TERM_DISCONT;
```

Comment:
Remember about DRY. You do this calculation 3 times(`days * PRICE_RER_DAY`), better to create variable